frontend/src/lib/components/modals/ConfirmDeleteModal.svelte

```diff
@@ -0,0 +1,56 @@
+<script context="module" lang="ts">
+  export type DeleteModalI18nShape = {
+    title: string;
+    submit: string;
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    enter_to_delete: {
+      label: string;
+      value: string;
+    };
+  };
+</script>
+
+<script lang="ts">
+  import Input from '$lib/forms/Input.svelte';
+  import { tScoped, type I18nShapeKey } from '$lib/i18n';
+  import { z } from 'zod';
+  import { FormModal } from '$lib/components/modals';
+  import type { FormModalResult, FormSubmitCallback } from '$lib/components/modals/FormModal.svelte';
+  import { TrashIcon } from '$lib/icons';
+
+  export let i18nScope: I18nShapeKey<DeleteModalI18nShape>;
+  let name: string;
+
+  export async function open(_name: string, onSubmit: FormSubmitCallback<Schema>): Promise<FormModalResult<Schema>> {
+    name = _name;
+    return await deletionFormModal.open(onSubmit);
+  }
+
+  const t = tScoped<DeleteModalI18nShape>(i18nScope);
+
+  const verify = z.object({
+    keyphrase: z.string().refine((value) => value.match(`^${$t('enter_to_delete.value')}$`)),
```

Comment:
hummmm. I suppose this will work? I just wondering if it might break somehow. I guess it's not critical if there's a weird failure on another language.

Reply:
I can think of what might go wrong. I agree it's interesting 😆